timelock-impl/src/main/java/com/palantir/atlasdb/timelock/paxos/ClientAwarePaxosLearner.java

```diff
@@ -33,7 +33,7 @@
 
 @Path("/" + PaxosTimeLockConstants.INTERNAL_NAMESPACE
         + "/" + PaxosTimeLockConstants.CLIENT_PAXOS_NAMESPACE
-        + "/{client: [a-zA-Z0-9_-]+}"
```

Comment:
Discussed offline - apparently this is some limitation with some of the 3rd party libraries that we use. Not blocking on this as it's enforced elsewhere